media/robots.txt

```diff
@@ -32,13 +32,24 @@ Disallow: /*docs*$styles
 Disallow: /*docs*$toc
 Disallow: /*docs*$move
 Disallow: /*docs*$quick-review
+Disallow: /*docs*$samples
 Disallow: /*docs*$revert
 Disallow: /*docs*$repair_breadcrumbs
 Disallow: /*docs*$delete
 Disallow: /*docs*$restore
 Disallow: /*docs*$purge
 Disallow: /*docs*$subscribe
+Disallow: /*docs*$subscribe_to_tree
 Disallow: /*docs*$vote
 Disallow: /*docs.json
-Disallow: /*preview-wiki-content
+Disallow: /*docs/ckeditor_config.js
+Disallow: /*feed*
+Disallow: /*files*
```

Comment:
This rule needs to be more precise at it will also block articles like https://developer.mozilla.org/en-US/docs/Using_files_from_web_applications